### PR TITLE
Update workflow documentation and examples

### PR DESCRIPTION
**Pull request by weierophinney**
- Use a standard `GITHUB_TOKEN` for the `Bump Changelog Version ..." step.
- Document why you might want to use a PAT with the "Release" step, and how to accomplish that.

**NO MILESTONE, NO RELEASE NEEDED**; this is just documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,26 @@ to your project or organization:
   command, because [changing default branch of a repository currently requires administrative token rights](https://developer.github.com/v3/repos/#update-a-repository).
   You can generate a token from your [personal access tokens page](https://github.com/settings/tokens/new).
 
+> ### Triggering release workflow events
```
Comment (Ocramius) [inline README.md]: Is the quoting intentional?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,26 @@ to your project or organization:
...
+> ```yaml
+>     - name: "Release"
+>       uses: "./"
+>       with:
+>         command-name: "laminas:automatic-releases:release"
+>       env:
+>         "GITHUB_TOKEN": ${{ secrets.ORGANIZATION_ADMIN_TOKEN }}
+> ```
+
 ## Usage
```
Comment (Ocramius) [inline README.md]: I think `Usage` should come before the `Triggering release workflow events`, which is a bit of a niche

File: README.md (hunk 1 above)
Comment (weierophinney) [reply README.md]: Wanted it to stand out. I can remove if you want.

File: README.md (hunk 2 above)
Comment (weierophinney) [reply README.md]: Reasonable - will push in a moment.

File: README.md (hunk 1 above)
Comment (Ocramius) [reply README.md]: Really looks like an edge case - semantically, quoting means something very different than "standing out" :D

**Review by Ocramius — APPROVED** [review]

**Pull request merged**